packages/graph/hash_graph/lib/graph/src/store/postgres/ontology/read.rs

```diff
@@ -46,3 +48,43 @@ where
             .change_context(QueryError)
     }
 }
+
+#[async_trait]
+impl<C: AsClient, T> crud::Read<'_, (), T> for PostgresStore<C>
+where
+    for<'de> T: OntologyDatabaseType + Deserialize<'de> + Send,
+{
+    type Output = Vec<T>;
+
+    async fn get(&self, _: ()) -> Result<Self::Output, QueryError> {
+        // SAFETY: We insert a table name here, but `T::table()` is only accessible from within this
```

Comment:
I don't really understand the SAFETY, there's no unsafe from what I can see so I'm not quite sure what this is about, and I don't fully understand the comment

Reply:
Generally, it should be avoided to create a SQL query using non-sql features like `format!`. In this case we know that `T::table()` is well defined for `OntologyDatabaseType`s, so this is safe to use. `OntologyDatabaseType` cannot be implemented outside of the `postgres` module.

Reply:
Can we update the comment to be a little clearer then? let me know if you want help on a call with wording